data/drumkits/TR808EmulationKit/drumkit.xml

```diff
@@ -2,12 +2,16 @@
 <drumkit_info xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://www.hydrogen-music.org/drumkit">
  <name>TR808EmulationKit</name>
  <author>ArtemioLabs [http://artemiolabs.com]</author>
- <info>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
-&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
-p, li { white-space: pre-wrap; }
-&lt;/style>&lt;/head>&lt;body style=" font-family:'Lucida Grande'; font-size:10pt; font-weight:400; font-style:normal;">
-&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">This drum kit contains 16 original sounds that were synthesized from scratch (using only basic synth waves, like sine, noise etc.). It simulates the sounds of the legendary Roland TR-808 Rhythm Composer. More kits from ArtemioLabs: http://artemiolabs.com/downloads/audio/hydrogen/&lt;/p>&lt;/body>&lt;/html></info>
+ <info>This drum kit contains 16 original sounds that were synthesized from scratch (using only basic synth waves, like sine, noise etc.). It simulates the sounds of the legendary Roland TR-808 Rhythm Composer. More kits from ArtemioLabs: http://artemiolabs.com/downloads/audio/hydrogen/</info>
  <license>GPL</license>
+ <image>Roland_TR-808_drum_machine.jpg</image>
```

Comment:
For the CC license: 
  **attribution** – You must attribute the work in the manner specified by the author or licensor (but not in any way that suggests that they endorse you or your use of the work).
  **share alike** – If you alter, transform, or build upon this work, you may distribute the resulting work only under the same or similar license to this one.

So if i understand those lines correctly, then including the image without any attribution and mentioning of the license would be a clear violation (if one would choose the CC license). In my opinion we should definitely not put any images with such restrictions in our default kits (if my interpretation of the license is correct).

I'm not sure sure about the GFDL, have to investigate further.